kratos/tests/test_force_and_torque_utils.py

```diff
@@ -0,0 +1,74 @@
+# Kratos Imports
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics
+
+# Standard Imports
+import pathlib
+
+def GetFullPathToFile(fileName):
+    return pathlib.Path(__file__).parent / fileName
```

Comment:
nice, I was wondering how to do this by using `pathlib`

Reply:
oh whoops, an ```absolute()``` is missing from there
```    return pathlib.Path(__file__).absolute().parent / fileName```